python/ray/tests/test_client_proxy.py

```diff
@@ -0,0 +1,88 @@
+import os
+import pickle
+import pytest
+import time
+
+import grpc
+import ray
+import ray.core.generated.ray_client_pb2 as ray_client_pb2
+from ray.job_config import JobConfig
+import ray.util.client.server.proxier as proxier
+
+
+def test_proxy_manager(shutdown_only):
```

Comment:
can we add documentation to explain what the tests are testing and how?

Reply:
Done!